src/accidental.ts

```diff
@@ -61,10 +62,10 @@ export class Accidental extends Modifier {
     // If there are no accidentals, no need to format their positions.
     if (!accidentals || accidentals.length === 0) return;
 
-    const noteheadAccidentalPadding = Tables.lookupMetric('Accidental.noteheadAccidentalPadding');
+    const noteheadAccidentalPadding = Metrics.lookupMetric('Accidental.noteheadAccidentalPadding');
```

Comment:
Can we rename to the shorter version:

`Metrics.lookup(...)`

Reply:
Or... how about

```
Metrics.get(...)
Metrics.set(...)
```

It's short and easy to understand. If we don't want to implement set for now, that's fine. But I can see a situation where a developer might want to temporarily change the metrics for a certain group of music symbols.

Reply:
Now MetricsDefaults is accesible, but a set would be easier. I have no idea how to code that. Could you do it on another PR?

Reply:
lookup changed to get.